tools/cldr-apps/js/package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "test": "exit 0",
     "lint": "exit 0",
-    "build": "webpack"
+    "build": "webpack",
+    "watch": "webpack watch"
```

Comment:
this turned out to be really handy!  I have `mvn … liberty:dev` in one window, and `npm run watch` in another. Instant reload of any file, whether .js  (fraction of seconds) or .java (10s of seconds!)

Reply:
When running that way, can you set breakpoints and step through the code?

Reply:
And is that with Eclipse, or without?

Reply:
No impact on front end experience. It means that the npm build happens automatically. This is just for the generation of bundle.js. Yes you can debug in eclipse or in browser.